sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContainerType.md

```diff
@@ -0,0 +1,69 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spocontainertype
+applicable: SharePoint Online
+title: Remove-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+ 
+# Remove-SPOContainerType
+ 
+## SYNOPSIS
+This cmdlet removes the containertype specified from the tenant.
+ 
+## SYNTAX
+ 
+### ParamSet1
+ 
+```powershell
+Remove-SPOContainerType -ContainerTypeId <ContainerTypeId>
+```
+ 
+## DESCRIPTION
+ 
+The Remove-SPOContainerType cmdlet deletes only the trial container type in the tenant. To delete a container type in trial status, you must remove all containers of the container type first, including from the deleted container collection. To remove containers, refer to Consuming Tenant Admin. Once all the containers are deleted, trial containertype can be deleted using the PowerShell cmdlet.
```

Comment:
>The Remove-SPOContainerType [](http://example.com/codeflow?start=0&length=27)

```suggestion
This
```

Reply:
Is the recommendation to remove the description added?

Reply:
It's about not repeating the cmdlet name everywhere, since the entire page is about that cmdlet. so instead of 

"The Remove-SPOContainerType cmdlet"

we'd just say

"This cmdlet"